### DISCUSSION TITLE
100C+ on MK52 Heater Bed. Instability

### DISCUSSION THREAD

**Discussion by confuzled-dev** [dz0ny/klipper-prusa-mk3s#29] [Q&A] [answered]
Has anyone here had heater_verify failures when printing with bed temps 100+?

I get pretty consistent failures where when the bed switches to 110 it fails I climb from 105 and begins to fall gradually until it throws an error.

I would love to see some feedback from others!!

**Comment by agravelot** [comment]
> Has anyone here had heater_verify failures when printing with bed temps 100+?
> 
> 
> 
> I get pretty consistent failures where when the bed switches to 110 it fails I climb from 105 and begins to fall gradually until it throws an error.
> 
> 
> 
> I would love to see some feedback from others!!

Can confirm this behavior. As workaround, i set my first layer and other layers at same temps.

**Reply by confuzled-dev** [reply]
What temps do you print asa at?

Also, what temps did you have trouble with? 105<>110<>failure

**Reply by agravelot** [reply]
> What temps do you print asa at?
> 
> 
> 
> Also, what temps did you have trouble with? 105<>110<>failure

I just keep at 100c for now. I don't print asa/abs that much on my prusa since it's not advisable.

**Comment by confuzled-dev** [comment]
Understood.

I have changed all my parts to asa so I'm not to concerned with part
damage/creep

**Comment by confuzled-dev** [comment]
I have found that this may be related to an ineffective PID tune. Because 110 is such a high temp for the bed, I am currently testing with the nozzle lowered to first layer height and fan set to 2nd layer speeds (50$ in my case). If this corrects the issue, I will report back and ask for testing from others prior to marking it as Solved

**Reply by confuzled-dev** [reply]
So it isnt an issue of PID. I can not maintain 100+ temp with the parts cooling fan turned on. I suppose my to option for testing are either with a pyrometer to see if the temps are even accurate (which might allow me to set a lower temp in klipper) or do flash back to stock and see if the issue goes away

**Comment by itsmiketorres** [comment]
Hey @dougshell, I'm also running into this same issue. I'm printing ASA in an enclosure with bed temps of 105-110C, and it's consistently triggering an error for ```MCU 'mcu' shutdown: ADC out of range``` aka the heater_verify failures you mentioned above. 

In my case the issue appears to be with the einsy board itself exceeding its specified max_temp of 70C. After adding a fan to the back of the enclosure temps have dropped to <50C and the printer is working again 🤞 

Try checking your logs to make sure your board is under the 70C. Initially, I was just looking at the extruder and heatbed, and ignoring the einsy board completely. Searching the logs I found ```einsy_board: temp=70.1```, which is beyond the default einsy board max_temp, explaining the shutdown.

Hope this helps!

**Comment by confuzled-dev** [comment]
I will check this out, however, this doesn't seem like the issue I was
having. My bed was not maintaining temp due to higher than normal fan temps
when the hotend was close to the bed. I did some testing, and with about a
5-10 minutes soak after reaching bed temp, I can maintain 105 with 40% fan
after the first layer.

If I increase the bed temp or increase the fan speed, I get failures again.

**Comment by confuzled-dev** [comment, marked as answer]
At least on my end, this is considered Solved. The printer struggles to maintain high temp when the parts fan is enabled. The closer the parts fan is to the unprotected build surface (no first layer established) the less fan can be tolerated. 

I have worked around this by ensuring that my preheat macro lifts the print head far enough from the build surface to encourage circulation but not negatively affect bed temp.

**Reply by b-fitzpatrick** [reply]
This may be relevant: https://github.com/dz0ny/klipper-prusa-mk3s/discussions/40
